packages/http/httpx/kiota_http/httpx_request_adapter.py

```diff
@@ -88,7 +88,7 @@ def __init__(
             http_client = KiotaClientFactory.create_with_default_middleware()
         self._http_client = http_client
         if not base_url:
-            base_url = ""
+            base_url = str(http_client.base_url)
```

Comment:
we should also check the client base url is not None, or we'll end up storing "None" in the field.